lnrpc/rest-annotations.yaml

```diff
@@ -41,6 +41,8 @@ http:
       get: "/v1/peers/subscribe"
     - selector: lnrpc.Lightning.GetInfo
       get: "/v1/getinfo"
+    - selector: lnrpc.Lightning.GetRecoveryInfo
+      get: "/v1/getrecoveryinfo"
```

Comment:
```suggestion
      get: "/v1/recoveryinfo"
```

Reply:
hmm...why the action `get` is removed? shouldn't this endpoint be similar to `getinfo`?

Reply:
Get is already implied by the request type, so it seems redundant. Not a blocker though.

Reply:
In that sense, yep I think the `get` is redundant. Just want to keep things uniform though.